pages/common/esptool.md

```diff
@@ -0,0 +1,12 @@
+# esptool
+
+> Espressif SoC serial bootloader utility.
+> More information: <https://docs.espressif.com/projects/esptool/en/latest/esp32/>.
+
+- Flash a firmware file to an ESP chip with given port and baud rate:
+
+`sudo esptool.py --port {{port}} --baud {{baud_rate}} write_flash 0x0 binary_esp8266.bin`
```

Comment:
Hi, Thanks for your contribution. I wasn't able to find the `--baud` flag in the documentation. What does it do?

Reply:
from esptool.py --help:

  --baud BAUD, -b BAUD  Serial port baud rate used when flashing/reading